app/pages/project/home/index.jsx

```diff
@@ -132,5 +133,12 @@ ProjectHomeContainer.propTypes = {
     researcher_quote: React.PropTypes.string
   }).isRequired,
   projectIsComplete: React.PropTypes.bool.isRequired,
-  splits: React.PropTypes.object
+  splits: React.PropTypes.object,
+  translation: React.PropTypes.shape({
```

Comment:
This PropType is failing with `translation` as `undefined` when testing with this project using the English default: https://translation-api.pfe-preview.zooniverse.org/projects/eatyourgreens/-project-testing-ground

Reply:
Not sure what's going on here. There's a default, empty translation set as the initial state in the translations ducks file.

Reply:
Is this the same issue we had before with some other props? They seemed to error before rendering...

Reply:
Aha! This warning is being thrown because there's no translation prop defined in `router.cjsx`. I'm not sure how to handle that.

Reply:
@simoneduca Yes! If you look at the stack trace, the calling context is the `RouterContext` component.

Reply:
Silly but can we not define the missing props in `router.cjsx`?

Reply:
@simoneduca How do you do that?

Reply:
Mm, that [doesn't seem to have a straight answer](https://github.com/ReactTraining/react-router/issues/1857#issuecomment-180879045).

Reply:
@simoneduca I know. I thought maybe you'd come across a way to pass default props in the routes definitions.

Reply:
Looks like required props are not recommended on top-level components.